helm_deploy/values-staging.yaml

```diff
@@ -17,6 +17,17 @@ generic-service:
     WHEREABOUTS_API_URL: https://whereabouts-api-dev.service.justice.gov.uk
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
+  namespace_secrets:
```

Comment:
will staging and dev end up listening on the same hmpps queue as staging might not be there on the env where the queues are configured?

Reply:
Yeha the fix for this is in CP!